spec/outputs/example_spec.rb

```diff
@@ -1 +1,20 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/example"
+require "logstash/codecs/plain"
+require "logstash/event"
+
+describe LogStash::Outputs::Example do
+  let(:sample_event) { LogStash::Event.new }
+
+  context "#receive" do
+    subject do
+      output = LogStash::Outputs::Example.new
+      output.register
+      output
+    end
+
+    it "returns hello world" do
+      expect(subject.receive(sample_event)).to eq("Event received")
```

Comment:
Not sure what is the most pedagogical point of view here, keeping in mind people can use this as an starting point to write good rspec tests. So ideas accepted.

What about writing this as something like:

```

describe "receive message" do

  subject { output.receive(sample_event) }

  it "returns an string" do
    expect(subject).to eq("Event received")
  end
end
```

this would make this test longer, but more clear that a good test have sections where different test scenarios are described. What do you think?